github-actions/start-release/build_docs.py

```diff
@@ -72,9 +72,14 @@ def escape_markdown(data):
 
 def get_app_json(app_json_dir):
     logging.info("Looking for app JSON in: %s", app_json_dir)
-    app_json_name = [f for f in os.listdir(app_json_dir)
-                     if f.endswith(".json")
-                     and "postman_collection" not in f][0]
+    app_json_name = ""
+    try:
+        app_json_name = [f for f in os.listdir(app_json_dir)
+                         if f.endswith(".json")
+                         and "postman_collection" not in f][0]
```

Comment:
I don't know where this is coming from, but... what if I have another json file in the app dir?

Reply:
There is a lot of pretty much this logic all over the place, I'd leave this as is but having a better way to tell which json is the json would be kewl

Reply:
I can add another argument in argparser containing name of a .json file, it would be set the same as dir name as default. With that user can generate readme even if they have .json with different name or they have multiple .json files. It would require changes in build_docks.py tho and I do not know if that is an issue.